src/Document/DocumentNormalizerTrait.php

```diff
@@ -84,7 +84,9 @@ protected function editableRelation(
                 [$contents->getId()],
                 array_map(
                     fn (Concrete $obj): int => $obj->getId(),
-                    $contents->getChildren([AbstractObject::OBJECT_TYPE_OBJECT])
+                    $contents
+                        ->getChildren([AbstractObject::OBJECT_TYPE_OBJECT])
+                        ->getData() ?? []
```

Comment:
Just curious, when does `getData()` from a Listing not return an array?

Reply:
The signature of `getData()` is `public function getData(): ?array;` and in some cases it returns the property `$data` directly which is defined as `protected ?array $data = null;` so there's a plausible chance of it returning `null`.